create/skeletons/theme/layouts/_partials/head/css.html

```diff
@@ -1,9 +1,9 @@
 {{- with resources.Get "css/main.css" }}
-  {{- if eq hugo.Environment "development" }}
-    <link rel="stylesheet" href="{{ .RelPermalink }}">
-  {{- else }}
+  {{- if hugo.IsProduction }}
```

Comment:
I suggest we keep the old conditional (but replace it with `hugo.IsDevelopment`).

I find that the most generally usable construct: 1. When running the server, no SRI etc, 2. When running in production or in staging/PR preview: SRI etc.